src/components/CvViewer/formatters/date.formatters.ts

```diff
@@ -0,0 +1,8 @@
+export const getDateFromTimeStampInDay = (timestamp?: number) => {
+  if (!timestamp) return '';
+  return new Date(timestamp).toLocaleDateString('ru-RU', {
```

Comment:
It's better to use here _en-US_ instead of _ru-RU_

Reply:
will do in next pr